dist/mention.js

```diff
@@ -145,7 +159,7 @@ angular.module('ui.mention').controller('uiMention', ["$element", "$scope", "$at
   this.decode = function () {
     var value = arguments.length <= 0 || arguments[0] === undefined ? ngModel.$modelValue : arguments[0];
 
-    return value ? value.replace(/@\[([\s\w]+):[0-9a-z-]+\]/gi, '$1') : '';
+    return value ? value.replace(this.decodePattern(), '$1') : '';
```

Comment:
Wouldn't it make more sense to make this a property instead of a function, similar to `this.pattern`?